x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -100,16 +100,18 @@ export const EnginesList: React.FC = () => {
       <EuiText size="s">
         <FormattedMessage
           id="xpack.enterpriseSearch.content.engines.enginesList.description"
-          defaultMessage="Showing {currentPage}-{size} of {total}"
+          defaultMessage="Showing {from}-{to} of {total}"
           values={{
-            currentPage: (
+            from: (
               <strong>
-                <FormattedNumber value={meta.from} />
+                <FormattedNumber value={meta.from + 1} />
               </strong>
             ),
-            size: (
+            to: (
               <strong>
-                <FormattedNumber value={meta.size} />
+                <FormattedNumber
+                  value={meta.from + meta.size < meta.total ? meta.from + meta.size : meta.total}
```

Comment:
question: if this logic is ever needed I feel like something has gone wrong at an earlier point in the data flow 🤔

Reply:
this just changes 10-20 of 15 to 10-15 of 15.

not really necessary, but looks slightly better.

I still wanted to abstract this whole section into a component b/c it does 11-11 of 11,  but i stopped myself.

Reply:
Wait so `meta.size` isn't actually the number of items returned, but the number of items requested? Argh

Reply:
Ah, but shouldn't this just be:
```
value={meta.from + (items?.length ?? 0)}
```

Reply:
that's cleaner :+1: